pkg/asset-manager-utils/test/RewardsAssetManager.test.ts

```diff
@@ -342,76 +325,128 @@ describe('Rewards Asset manager', function () {
   });
 
   describe('rebalance', () => {
-    context('when pool is above target investment level', () => {
-      const poolConfig = {
-        targetPercentage: fp(0.5),
-        upperCriticalPercentage: fp(1),
-        lowerCriticalPercentage: fp(0.1),
-      };
+    function itRebalancesCorrectly(force: boolean) {
+      let poolConfig: PoolConfig;
 
       sharedBeforeEach(async () => {
-        const poolController = lp; // TODO
-        await assetManager.connect(poolController).setPoolConfig(poolId, poolConfig);
-
-        const { poolCash } = await assetManager.getPoolBalances(poolId);
-        await tokens.DAI.mint(assetManager.address, poolCash.mul(101).div(100));
+        poolConfig = await assetManager.getPoolConfig(poolId);
+      });
```

Comment:
This is only used in the 'transfers the expected tokens' test, I'd move it there :shrug:

Reply:
True, I was thinking that we'd be reusing it in other tests in that block but it's not really worth the `sharedBeforeEach`